data/Mongolia/Assembly/ep-popolo-v1.0.json

```diff
@@ -4934,7 +4949,7 @@
       "sort_name": "Namdag BATTSEREG"
     },
     {
-      "birth_date": "1963-03-03",
+      "birth_date": "1963",
```

Comment:
I've removed the less specific one, so this should flow through again on next build